tests/utils.py

```diff
@@ -39,7 +39,13 @@ def skip_if_moftab_regenerated():
     """
 
     pywbem = import_installed('pywbem')  # noqa: E402
-    from pywbem import mofparsetab, moflextab
+
+    try:
+        # If you cannot write to the directory you can't make these files, thus they don't exist
+        # and you cannot import them.
+        from pywbem import mofparsetab, moflextab
+    except ImportError:
+        pytest.skip("We have no mofparsetab or moflextab to import!")
```

Comment:
They should always exist: when testing against the local directory, the makefile ensures they get created. when testing against an installed version, the installed version needs to contain them. with a pypi package that is the case. with a distro provided os level package that ought to be the case as well.

Reply:
Nevertheless, we can leave this in as an additional safeguard. Accepted.